pymc/tests/test_mixture.py

```diff
@@ -19,11 +19,15 @@
 import pytest
 import scipy.stats as st
 
+from aeppl.transforms import IntervalTransform, LogTransform
+from aeppl.transforms import Simplex as SimplexTransform
 from aesara import tensor as at
+from aesara.tensor import TensorVariable
 from aesara.tensor.random.op import RandomVariable
 from numpy.testing import assert_allclose
 from scipy.special import logsumexp
 
+from pymc import Triangular
```

Comment:
Shouldn’t this import be under from distributions instead?

Reply:
Yes, thanks